bin/aws/ssm_consul_server.sh

```diff
@@ -1,13 +1,39 @@
 #!/usr/bin/env bash
 
+# This script sets up port forwarding between an arbitrary consul server and your local computer.
+# This is a temporary workaround until we set up a VPN
+# This will allow you to view the consul UI and interact with consul in the cli
+#
+# Input (optional): Local port to forward to. Defaults to port 8500
+
 set -euo pipefail
 
-SERVERS=$(aws ec2 describe-instances --filter Name=tag:Name,Values="Consul Server")
-CONSUL_SERVER_INSTANCE_ID=$(echo "${SERVERS}" | jq -r .Reservations[0].Instances[0].InstanceId)
+LOCAL_PORT_TO_FORWARD_TO="${1:-8500}"
+
+SSM_PARAMETERS=$(
+    cat << EOF
+{
+  "portNumber": ["8500"],
+  "localPortNumber": ["${LOCAL_PORT_TO_FORWARD_TO}"]
+}
+EOF
+)
+
+echo "Connecting to a consul server in AWS PROFILE: ${AWS_PROFILE:-"No AWS Profile is set, please run export AWS_PROFILE=foo"} on port 8500 and forwarding to ${LOCAL_PORT_TO_FORWARD_TO}"
```

Comment:
I think this is going to generate a weird message when the variable is unset.

Reply:
Switching this to a if -z check